lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -593,6 +593,9 @@ instance EncodeAddress t => ToText (AddressAmount t) where
     toText (AddressAmount (ApiT addr, proxy) coins) =
         toText coins <> "@" <> encodeAddress proxy addr
 
+instance FromText PostExternalTransactionData where
+    fromText = pure . PostExternalTransactionData . T.encodeUtf8
```

Comment:
This is where a conversion from hex should happen, if anywhere.

Reply:
done